pkgs/development/python-modules/soco/default.nix

```diff
@@ -1,29 +1,54 @@
-{ lib, buildPythonPackage, fetchPypi, xmltodict, requests
-, toml
-
-# Test dependencies
-, pytest, pytestcov, coveralls, pylint, flake8, graphviz, mock, sphinx
+{ buildPythonPackage
+, coveralls
+, fetchFromGitHub
+, flake8
+, graphviz
+, lib
+, mock
+, pytestCheckHook
+, requests
+, sphinx
 , sphinx_rtd_theme
+, toml
+, xmltodict
 }:
 
 buildPythonPackage rec {
   pname = "soco";
   version = "0.20";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "929d4fae20b32efc08bb9985798c592aa7268162885541513eddbff0a757418f";
+  src = fetchFromGitHub {
+    owner = "SoCo";
+    repo = "SoCo";
+    rev = "v${version}";
+    sha256 = "0p87aw7wxgdjz0m0nqqcfvbn24hlbq1hh1zxdq2c0k2jcbmaj8zc";
   };
 
+  # N.B. These should probably be removed on the next release, assuming by then
+  # their versions have caught up with nixpkgs.
+  # I say probably because I don't think there's any benefit for us to be
+  # running linting and coverage tests, but alas
+  postPatch = ''
+    sed -i "/black/d" ./requirements-dev.txt
+    sed -i "/pylint/d" ./requirements-dev.txt
+    sed -i "/pytest-cov/d" ./requirements-dev.txt
+  '';
+
   propagatedBuildInputs = [ xmltodict requests toml ];
```

Comment:
reformat this to match

Reply:
What do you mean? Match what?

Reply:
if you mean so that it looks
```
sed -i foooo ./bar
sed -i foo     ./bar
sed -i fooo   ./bar
```
then that's a stylistic choice I dislike and would rather not do.

Reply:
Reformat `propagatedBuildInputs` in the  same fashion you did for `checkInputs`. Sorry that wasn't clear.

Reply:
Oh, I see, done!